### ISSUE TITLE
Cut and paste between diagrams

### ISSUE THREAD

**Issue by sanyimano** [juzzlin/Heimer#247]
Hi!
The cut/paste excellently works in the same diagram.
Az a first step I prepare a scheme, and draw the details on individual diagrams.
Than I build the scheme again from the detailled diagrams.
Is it possible to make the use of cut/paste between diagrams if the job is not too complicated?

best regards

**Comment by sanyimano** [comment]
A solution came up to my mind. What about this:
Export the selected nodes with the accompanied edges to intermediate "graph" xml file
On the paste side with adjusting the node numbering and the paste point the "graph" could be imported.

**Comment by juzzlin** [comment]
> A solution came up to my mind. What about this: Export the selected nodes with the accompanied edges to intermediate "graph" xml file On the paste side with adjusting the node numbering and the paste point the "graph" could be imported.

Yes, I can make this happen. I'm really busy ATM with my private life, but I'll get back to this.

**Comment by sanyimano** [comment]
Thanks in advance
